scripts/dev/generate_epJSON_schema/generate_epJSON_schema.py

```diff
@@ -62,7 +62,7 @@
 
 source_dir_path = sys.argv[1]
 data = idd_parser.Data()
-with open(path.join(source_dir_path, 'idd', 'Energy+.idd.in'), 'r') as f:
+with open(path.join(source_dir_path, 'Energy+.idd'), 'r') as f:
```

Comment:
And here inside generate_schema, the only argument is the directory to operate inside.  It finds the IDD in there, and creates the schema file in there.